sdk/appconfiguration/azure-data-appconfiguration/samples/appconfig_basic_operation.cpp

```diff
@@ -74,6 +74,181 @@ static void RetreiveLabels(ConfigurationClient& configurationClient)
 #endif
 }
 
+// Retrieve a snapshot
+static void RetrieveSnapshot(ConfigurationClient& configurationClient)
+{
+  // Current
+
+  {
+    Azure::Response<GetSnapshotResult> getSnapshotResult
+        = configurationClient.GetSnapshot("snapshot-name", "accept");
```

Comment:
So, I see in many cases, the main (and often the only) thing that needs to be fixed so that we move to the desired syntax is this accept parameter, correct?

Reply:
Yes, for this method, that's the only change left from an API design perspective. now that returning `GetSnapshotResult` vs `Snapshot` has been fixed with your recent changes (of excluding response headers from models). The other aspect that comes up often related to the `accept` parameter is the `content-type` input param.

That said, it's possible we discover some implementation correctness issues when we test the various `GetSnapshotOptions` values passed in. I haven't seen any issues yet.

In other cases (like Archive/Recover below), a bit more hand-massaging is required.